gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/computer/traversal/step/VertexComputing.java

```diff
@@ -53,7 +53,19 @@
      * @param memory the {@link Memory} from the previous OLAP job if it exists, else its an empty memory structure.
      * @return the generated vertex program instance.
      */
-    public VertexProgram generateProgram(final Graph graph, final Memory memory);
+    @Deprecated
```

Comment:
As noted above, the javadoc needs some entries given the deprecation.

Reply:
Done.